docs/development/how_to_add_a_route.md

```diff
@@ -0,0 +1,175 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+The goal of this document is to cover all the necessary steps to add an API route to Parsec.
+
+In summary:
+1. Add the schema describing the new API route
+2. Generate binding code and serialization tests
+3. Implement server-side code (In-memory & PostgreSQL) and tests
+4. Implement client-side code and tests
+
+> Note that you will need a working development environment. See (./README.md).
+## Schema generation
```

Comment:
There is multiple markdown syntax error in the file, here is a patch to fix them

```
$ git apply how_to_add_a_route.md.patch.txt
```


[how_to_add_a_route.md.patch.txt](https://github.com/user-attachments/files/15903059/how_to_add_a_route.md.patch.txt)